src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -2863,6 +2863,11 @@
 								</VisualState>
 								<VisualState x:Name="ClosedCompactRight">
 									<Storyboard>
+										<xamarin:ObjectAnimationUsingKeyFrames Storyboard.TargetProperty="Clip"
```

Comment:
Why is this necessary? I assume it's a workaround, but what for?

Reply:
The same solution is for the left clip, but it was probably forgotten for the right clip. UWP animates `RectangleGeometry.Transform` instead, which is not yet supported

Reply:
Fair enough. Can you add a comment explaining the reason for it please - ideally linking to an issue for the `RectangleGeometry.Transform` limitation

Reply:
It is actually documented here - https://github.com/unoplatform/uno/blob/8dcc6f2d3aa5d57fd3ee96efe5fe73855a45a0da/src/Uno.UI/UI/Xaml/Controls/SplitView/SplitViewTemplateSettings.cs#L36

Reply:
Added #3238 to track this